resumes-api/src/api/Jalasoft.TeamUp.Resumes.API/Startup.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Jalasoft.TeamUp.Resumes.API;
```

Comment:
please put these inside the namespace

Reply:
if I move the statement inside the namespace, assembly doesn´t work because must go outside any definition